taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+        </li>
+        <li>
+          <a target="_blank" href="https://frontera-portal.tacc.utexas.edu/user-guide/files/#transferring-files-with-rsync">Frontera User Guide: Transferring Files with <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides">TACC User Guides (for each system)</a>
+          <small>(select the relevant system for this portal, then search "<code>scp</code>")</small>
+        </li>
+      </ul>
+
+      <h4 id="sftp_transfer">Transfer Data with <code>sftp</code></h4>
+
+      <p>
+        <code>sftp</code>
+        is a file transfer program that allows you to interactively navigate between your local file system and the remote secure system. To transfer a file (ex. <code>my_file.txt</code>) to the remote secure system via <code>sftp</code>, open a terminal on your local computer and navigate to the path where your data file is located.&nbsp;
+      </p>
+      <pre style="max-width: 500px"><code><u>localhost$ </u>cd Documents/portal-data/</code></pre>
+
+      <p>
+        Assuming your TACC username is <code>jdoe</code> and you are affiliated with UT Austin, a <code>sftp</code> transfer that pushes
+        <code>my_file.txt</code>
+        from your local computer to the remote secure system would look like this:
+      </p>
+      <pre style="max-width: 1500px"><code><u>localhost$ </u>sftp jdoe@<kbd>host</kbd>:<kbd>/transfer/directory/path</kbd>
+<samp>Password:
+TACC Token Code:
+Connected to <var>host</var>.
+Changing to:
+  <var>/transfer/directory/path</var></samp>
+<u>sftp&gt;</u></code></pre>
+
+      <blockquote>
+        <p>If you have not done so already, enter this command in your terminal,
+          replacing the TACC username and your individualized transfer directory name appropriately.</p>
+      </blockquote>
+
+      <p>
+        At this point, you have logged in to the remote secure system and been redirected to your transfer directory. To confirm your location on the server, enter the following command:
+      </p>
+
+      <pre style="max-width: 1000px"><code><u>sftp&gt; </u>pwd
+<samp>Remote working directory:
+  <var>/transfer/directory/path</var></samp></code></pre>
+
+      <p>To list the files currently in your transfer directory:</p>
+
+      <pre style="max-width: 500px"><code><u>sftp&gt; </u>ls
+<samp>utaustin_dir.txt<samp></code></pre>
+
+      <p>
+        To list the files currently in your <em>local</em> directory (ex.
+        <code>/Users/myname/Documents/portal-data/</code>):
+      </p>
+
+      <pre style="max-width: 500px"><code><u>sftp&gt; </u>lls
+<samp>my_file.txt<samp></code></pre>
+
+      <blockquote>
+        <p>
+          Note: The leading <em>l</em> in the <code>lls</code> command denotes that you are listing the contents of your <em>local</em> working directory.
```

Comment:
SUGGESTION: This is confusing because the italicized l renders like a forward slash. Better to use <code>l</code> here.